commit 984f94afb974765716b75275c45fd0280b397f98
Author: Boris Pek <tehnick-8@yandex.ru>
Date:   Sat May 12 18:55:44 2012 +0300

    Qt UI: fixed crash at launch time
    (problem was only in builds with gcc 4.7)
    [Thanks to Pino Toscano]

diff --git a/eiskaltdcpp-qt/src/EiskaltApp.h b/eiskaltdcpp-qt/src/EiskaltApp.h
index ad775d4..7c854b2 100644
--- a/eiskaltdcpp-qt/src/EiskaltApp.h
+++ b/eiskaltdcpp-qt/src/EiskaltApp.h
@@ -87,7 +87,7 @@ private:
 class EiskaltApp: public QtSingleCoreApplication {
 Q_OBJECT
 public:
-    EiskaltApp(int argc, char *argv[], const QString& uniqKey): QtSingleCoreApplication(argc, argv, uniqKey)
+    EiskaltApp(int &argc, char *argv[], const QString& uniqKey): QtSingleCoreApplication(argc, argv, uniqKey)
     {
         installEventFilter(&ef);
     }
diff --git a/eiskaltdcpp-qt/src/EiskaltApp_haiku.h b/eiskaltdcpp-qt/src/EiskaltApp_haiku.h
index 3b94bea..7a34180 100644
--- a/eiskaltdcpp-qt/src/EiskaltApp_haiku.h
+++ b/eiskaltdcpp-qt/src/EiskaltApp_haiku.h
@@ -87,7 +87,7 @@ private:
 class EiskaltApp: public QApplication {
 Q_OBJECT
 public:
-    EiskaltApp(int argc, char *argv[], const QString& uniqKey): QApplication(argc, argv)
+    EiskaltApp(int &argc, char *argv[], const QString& uniqKey): QApplication(argc, argv)
     {
         installEventFilter(&ef);
     }
